src/azure/monitor/opentelemetry/distro/distro.py

```diff
@@ -0,0 +1,15 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from os import environ
+from opentelemetry.instrumentation.distro import BaseDistro
+from opentelemetry.environment_variables import OTEL_LOGS_EXPORTER, OTEL_METRICS_EXPORTER, OTEL_TRACES_EXPORTER
+
+class AzureMonitorDistro(BaseDistro):
+    def _configure(self, **kwargs):
+        environ.setdefault(OTEL_LOGS_EXPORTER, "otlp_proto_grpc")
```

Comment:
Shouldn't these use the azure monitor exporters?

Reply:
Looks like I used the wrong branch. Will fix.

Reply:
Done. Thanks for catching.